tests/checks/indent.fish

```diff
@@ -487,6 +487,54 @@ level 2 } }
     # CHECK: {{^\}$}}
 } | $fish_indent
 
+echo 'false or {
+    echo hi
+}' | $fish_indent
+# CHECK: false or {
+# CHECK: {{^    }}echo hi
+# CHECK: {{^}}}
+
+echo 'true and {
+    echo hi
+}' | $fish_indent
+# CHECK: true and {
```

Comment:
can you squash your commits into one?
We don't want unnecessary noise in the commit history.